src/modules/settings/AppSettings.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import { AsyncStorage } from 'react-native'
+import { mapValues, toPairs } from 'lodash/object'
+import { fromPairs } from 'lodash/array'
+
+export type SettingsType = {|
+  errorTracking: boolean | null,
+  test: boolean | null
+|}
+
+const e2eSettings = {
+  errorTracking: false,
+  test: false
+}
+
+export const defaultSettings: SettingsType = (__DEV__ || process.env.E2E_TEST_IDS) ? e2eSettings : {
+  errorTracking: null,
+  test: false
+}
+
+class AppSettings {
+  asyncStorage: AsyncStorage
+
+  constructor (asyncStorage: AsyncStorage = AsyncStorage) {
+    this.asyncStorage = asyncStorage
+  }
+
+  async loadSettings (): Promise<SettingsType> {
+    const settingsKeys = Object.keys(defaultSettings)
+    const settingsArray = await AsyncStorage.multiGet(settingsKeys)
```

Comment:
either use `this.asyncStorage.multiGet`  or remove `asyncStorage` property from this class...